### PR TITLE
New Rule Proj0400: Explicit IsPublishable

### PR DESCRIPTION
**Pull request by CptWesley**
**Proj0400** Define the project publishability explicitly

### CODE REVIEW

File: specs/DotNetProjectFile.Analyzers.Specs/Rules/MS_Build/Define_is_publishable.cs [hunk 1]
```diff
@@ -0,0 +1,21 @@
+﻿namespace Rules.MS_Build.Define_is_publishable;
+
+public class Reports
+{
+    [Test]
+    public void on_no_is_publishable()
+       => new DefineIsPublishable()
+       .ForProject("OnlyTargetFramework.cs")
```
Comment (Corniel) [inline specs/DotNetProjectFile.Analyzers.Specs/Rules/MS_Build/Define_is_publishable.cs:8]: Use EmptyProject instead.

File: rules/Proj0400.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# Proj0400: Define the project publishability explicitly
+To prevent confusion, explicitly define the
+`<IsPublishable>` node.
+
```
Comment (Corniel) [inline rules/Proj0400.md:4]: Add a link to MS documentation to explain what publishable is about, as this has meaning for the rules to come.

**Review by Corniel — CHANGES_REQUESTED** [review]

File: rules/Proj0400.md (hunk 2 above)
Comment (Corniel) [reply rules/Proj0400.md:4]: May be this one: https://learn.microsoft.com/en-us/dotnet/core/tools/dotnet-publish

File: specs/DotNetProjectFile.Analyzers.Specs/Rules/MS_Build/Define_is_publishable.cs (hunk 1 above)
Comment (CptWesley) [reply specs/DotNetProjectFile.Analyzers.Specs/Rules/MS_Build/Define_is_publishable.cs]: ✔️

File: rules/Proj0400.md (hunk 2 above)
Comment (CptWesley) [reply rules/Proj0400.md:15]: ✔️

**Review by Corniel — APPROVED** [review]
LGTM

**Pull request merged**